core/src/main/kotlin/org/evomaster/core/problem/graphql/service/GraphqlSampler.kt

```diff
@@ -0,0 +1,121 @@
+package org.evomaster.core.problem.graphql.service
+
+import graphql.schema.idl.TypeDefinitionRegistry
+import graphql.schema.idl.SchemaParser
+import graphql.introspection.IntrospectionResultToSchema
+import graphql.introspection.Introspection
+import graphql.parser.Parser
+import graphql.language.Document
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import java.util.Map
+import graphql.introspection.IntrospectionQuery
+import java.net.URLEncoder
+import com.google.inject.Inject
+import org.evomaster.client.java.controller.api.dto.SutInfoDto
+import org.evomaster.core.EMConfig
+import org.evomaster.core.problem.graphql.GraphqlAction
+import org.evomaster.core.problem.graphql.GraphqlIndividual
+import org.evomaster.core.remote.SutProblemException
+import org.evomaster.core.remote.service.RemoteController
+import org.evomaster.core.search.gene.ObjectGene
+import org.evomaster.core.search.service.Sampler
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.net.ConnectException
+import javax.annotation.PostConstruct
+import javax.ws.rs.client.ClientBuilder
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+
+class GraphqlSampler : Sampler<GraphqlIndividual>() {
+
+    companion object {
+        private val log: Logger = LoggerFactory.getLogger(GraphqlSampler::class.java)
+    }
+
+    @Inject
+    private lateinit var rc: RemoteController
+
+    @Inject
+    private lateinit var configuration: EMConfig
+
+    private val adHocInitialIndividuals: MutableList<GraphqlAction> = mutableListOf()
+
+    private val modelCluster: MutableMap<String, ObjectGene> = mutableMapOf()
+
+    @PostConstruct
+    private fun initialize() {
+
+        GraphqlSampler.log.debug("Initializing {}", GraphqlSampler::class.simpleName)
```

Comment:
why are u using `GraphqlSampler.log` instead of just writing `log` ?

Reply:
Oh ! Yes you're right It should be log.